part-3/pk-sim-compounds-definition-and-work-flow.md

```diff
@@ -278,54 +274,58 @@ Protein Binding Partners
 {% endhint %}
 
 #### Transport & Excretion
 
-Drug transport across endothelial, epithelial or cellular barriers is responsible for distribution and renal or biliary elimination of a compound. Different experimental approaches are available either to determine rate constants or organ clearances. Depending on the experimental data available you can define different process types for your compound in the **Transport Proteins** branch, the **Renal Clearances** branch or the **Biliary Clearance** branch.
+Drug transport across endothelial, epithelia, or cellular barriers is responsible for the distribution and renal or biliary elimination of a compound. Different experimental approaches are available either to determine rate constants or organ clearances. Depending on the experimental data available you can define different process types for your compound in the **Transport Proteins**, the **Renal Clearances**, or the **Biliary Clearance** section.
 
 **Transport Proteins**
 
 | Process Type                                                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Necessary Input Parameter                                                          |
 | -------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
-| Intrinsic active transport - Michaelis-Menten                  | <p>A transporter-mediated unidirectional transfer of a compound across a cellular membrane is described as saturable process following the Michaelis-Menten type kinetics. The specific Vmax value used in the simulation (referring to the cellular volume) is scaled from the Vmax in liver tissue using the following calculation parameters:<br>• Fraction interstitial (liver)<br>• Fraction intracellular (liver)<br>The default value for Km is 1 μmol/l. It may be manually changed.</p> | Km; Vmax (measured with liver tissue or fitted)                                    |
```

Comment:
The "unidirectional" is not correct, right? We have bi-directional transports that can be coupled with this transport type?